ivre/db/mongo.py

```diff
@@ -904,6 +904,28 @@ def searchfile(self, fname):
             output=fname,
         )
 
+    def searchsmbshares(self, access=None):
+        """Filter SMB shares with anonymous `access` (default: either
+        read or write, accepted values 'r', 'w', 'rw')
+
+        """
+        if access is None:
+            access = ''
+        access = access.lower()
+        if 'w' in access:
```

Comment:
Nice one! I was looking for a good way to implement that but couldn't find one. The dictionnary is an excellent idea!